Big-Spender/readme.md

```diff
@@ -112,7 +112,7 @@ INSERT YOUR QUERY HERE
 **You:** Then you need an extra clause. Here's the query:
 
 ```sql
-CREATE YOUR QUERY HERE
+select s.supplier, sp.date, sum(sp.amount) as total from spends sp inner join suppliers s on (s.id = sp.supplier_id) where sp.date in ('2021-03-01', '2021-04-01') group by s.supplier, sp.amount, sp.date;
```

Comment:
again, why grouping by amount here? You are already using sum of the amount which is an aggregate function. 

nice use of IN